src/AstProfileUse.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2018, The Souffle Developers. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file AstProfileUse.cpp
+ *
+ * Implements an AstAnalysis that provides profile information
+ * from a profile log file for profile-guided optimisations.
+ *
+ ***********************************************************************/
+
+#include "AstProfileUse.h"
+#include "AstClause.h"
```

Comment:
Remove the unused Ast headers, as well as Util.h, algorithm, iterator, and set. Add in profile/Reader.h and stdexcept

Reply:
Fixed.